Cargo.toml

```diff
@@ -53,7 +53,7 @@ js-sys = { version = "0.3", optional = true }       # contains FFI bindings for
 windows-targets = { version = "0.52", optional = true }
 
 [target.'cfg(windows)'.dev-dependencies]
-windows-bindgen = { version = "0.52" }
+windows-bindgen = { version = "0.53" } # The MSRV of its windows-metatada 0.53 dependency is 1.70
```

Comment:
Since this is a dev-dependency, why is this comment useful?

Reply:
Because the version requirement bit me once :smile:.
Just serving as documentation of the minimum rust version we need to do development on chrono on Windows.